scripts/issues.coffee

```diff
@@ -0,0 +1,43 @@
+# Description:
+#   Create GitHub issues for CH beheer and PC.COM. Please be concise.
+#
+# Dependencies:
+#   octonode
+#
+# Configuration:
+#   HUBOT_GITHUB_TOKEN = <accessToken>
+#
+# Commands:
+#   @pccom <query> - Creates a new issue and tags it 'pccom'
+#   @beheer <query> - Creates a new issue and tags it 'beheer'
+
+github = require 'octonode'
+
+githubToken = process.env.HUBOT_GITHUB_TOKEN;
+
+issue = null
+
+module.exports = (robot) ->
+  unless githubToken?
+    robot.logger.error "Missing HUBOT_GITHUB_TOKEN in environment"
+    return
+  robot.listen(
+    (message) ->
+      match = message.text?.match /^@(pccom|beheer) (?:me )?(.*)/i
```

Comment:
Hmm wanneer ik een random bericht stuur is `message.text` ineens undefined, behalve bij een bericht dat voldoet aan het pattern (tot nu toe tenminste).